### PR TITLE
{tools}[GCCcore/10.2.0] MUSCLE3 v0.7.0, protobuf v3.10.0, ... w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MUSCLE3-Python/MUSCLE3-Python-0.5.0-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+homepage = 'https://muscle3.readthedocs.io/en/latest/index.html'
+
+description = """MUSCLE3 allows connecting multiple simulation models
+ together into a multiscale simulation. Simulation models can be as
+ simple as a single Python file, or as complex as a combination of
+ multiple separate simulation codes written in C++ or Fortran, and
+ running on an HPC machine."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```
Comment (verdurin) [inline easybuild/easyconfigs/m/MUSCLE3-Python/MUSCLE3-Python-0.5.0-GCCcore-10.2.0.eb:14]: @SimonPinches missing build deep for `binutils`, as added to the others.

File: easybuild/easyconfigs/m/MUSCLE3-Python/MUSCLE3-Python-0.5.0-GCCcore-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+ multiple separate simulation codes written in C++ or Fortran, and
+ running on an HPC machine."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/multiscale/muscle3/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['abdbad412929d16a303b7e2909d2f1267857bc33514339c1c6f0b89dff2f5b4f']
+
+dependencies = [
```
Comment (verdurin) [inline easybuild/easyconfigs/m/MUSCLE3-Python/MUSCLE3-Python-0.5.0-GCCcore-11.3.0.eb:20]: Missing `binutils` build dep as above.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by akesandgren** [comment]
There shouldn't be any need for a separate MUSCLE3-Python package. The MUSCLE3 package should build/install all parts.
Not sure it will be easy, but worst case is that it would need its own easyblock.

File: easybuild/easyconfigs/m/MUSCLE3-Python/MUSCLE3-Python-0.5.0-GCCcore-10.2.0.eb (hunk 1 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/m/MUSCLE3-Python/MUSCLE3-Python-0.5.0-GCCcore-10.2.0.eb]: @verdurin Changed to use foss toolchain so not needed now.

File: easybuild/easyconfigs/m/MUSCLE3-Python/MUSCLE3-Python-0.5.0-GCCcore-11.3.0.eb (hunk 2 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/m/MUSCLE3-Python/MUSCLE3-Python-0.5.0-GCCcore-11.3.0.eb]: @verdurin Changed to use foss toolchain so not needed now.

**Comment by SimonPinches** [comment]
@akesandgren I agree that one MUSCLE3 module would be nice but I'm not sure how to do this and it's probably beyond the amount of time I have available :-(  Do you have any good examples I could learn from or other suggestions?

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 12 out of 50 (20 easyconfigs in total)
centos7.lan - Linux CentOS Linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/890e649931805c9e7f06a3c769238851 for a full test report.

**Comment by boegel** [comment]
closing this since `GCCcore/10.2.0` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**